src/flp.rs

```diff
@@ -545,6 +552,21 @@ pub trait Type: Sized + Eq + Clone + Debug {
     }
 }
 
+/// A type which supports adding noise to aggregate shares for Server Differential Privacy.
+#[cfg(feature = "experimental")]
+pub trait TypeWithNoise<S>: Type
+where
+    S: DifferentialPrivacyStrategy,
+{
+    /// Optionally add noise to the aggregate share to obtain differential privacy.
```

Comment:
IIUC the expected effect of this call is that it always adds noise.
```suggestion
    /// Add noise to the aggregate share to obtain differential privacy.
```

Reply:
Done.